openhtf/__init__.py

```diff
@@ -162,16 +164,26 @@ def plug_type_map(self):
     return plug_type_map
 
 
+def CreateArgParser():
+  """Creates an argparse.ArgumentParser for parsing command line flags."""
+  parser = argparse.ArgumentParser('OpenHTF-based testing', parents=[
+      conf.ARG_PARSER, user_input.ARG_PARSER, phase_executor.ARG_PARSER,
+      logs.ARG_PARSER])
+  return parser
+
+
 @functions.RunOnce
```

Comment:
the use of RunOnce with functions that take arguments scares me a little, because the arguments are silently ignored.  not sure what the best fix is for this...